e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -3806,3 +3802,98 @@ describe("issue 35852", () => {
     });
   }
 });
+
+describe("issue 49052", { tags: ["@external"] }, () => {
```

Comment:
49052 is this PR :see_no_evil: 

```suggestion
describe("issue 45670", { tags: ["@external"] }, () => {
```

Reply:
Thanks for catching this)